client/src/components/patientData.tsx

```diff
@@ -1,11 +1,34 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import { Button } from '@cmsgov/design-system';
 import Records from './records'
+import axios from 'axios';
 import chart from '../images/who-charted.png'
+import Settings from './settings';
+import { Authorization } from '../types/authorization';
+import { SettingsType } from '../types/settings';
 
 export default function PatientData({ }) {
     const [header, setHeader] = useState('Add your Medicare data');
     const [show, setShow] = useState(true);     
+    const [authToken, setAuthToken] = useState<Authorization | undefined>(undefined);
```

Comment:
```suggestion
    const [authToken, setAuthToken] = useState<Authorization?>();
```

lets try this

Reply:
This didn't work and I couldn't resolve the error.  Leaving that way for now.

Reply:
@nbragdon any other suggestions?  I think we will be ok leaving it as is, correct?